### DISCUSSION TITLE
@diff or @PR (Similar to cursor)

### DISCUSSION THREAD

**Discussion by jonit-dev** [cline/cline#621] [Feature Requests]
A way to refer specifically to the current diff (last commit) or whole PR

**Comment by nrajasiddharthan** [comment]
I ran a `git show <commit-id> > diff.txt` And referenced the file in the plugin

**Reply by rubslopes** [reply]
Alternately, you could just ask Cline to "check the last commit diff"

**Reply by KevinPartingtonTR** [reply]
I believe this now exists. You can reference a git commit or `@git-changes` for the working tree diff.
